### PR TITLE
Fix carbon dashboard UI issues

### LINKED ISSUE #1515: [carbon dashboard]  Align css variables with figma
CSS variables are not aligned with figma.

Figma: https://www.figma.com/file/mXI4SzNNmoefYnOdcbXZcO/Carbon-Dashboard?node-id=2638%3A14363&mode=dev
Variables: https://github.com/KlimaDAO/klimadao/blob/staging/carbonmark-data/app/%5Blocale%5D/globals.css

### PR DESCRIPTION
**Pull request by sprrwhwk**
## Related Ticket

Resolves #1515 

## Checklist

- [x] I have run `npm run build-all` without errors
- [x] I have formatted JS and TS files with `npm run format-all`

### CODE REVIEW

File: carbon/app/[locale]/off-chain-vs-on-chain/page.tsx [hunk 1]
```diff
@@ -15,8 +15,8 @@ export default function OffVsOnChainPage() {
       <PageHeader title={t`Off vs On-chain carbon`} />
       <div className={layout.cardStackedRows}>
         <div className={`${layout.cardRow} ${layout.prioritizeFirstCard}`}>
-          <VerraCreditsBreakdownCard />
-          <TokenizedCreditsByBridgeCard />
+          <VerraCreditsBreakdownCard className={layout.zIndexSeven} />
+          <TokenizedCreditsByBridgeCard className={layout.zIndexSix} />
```
Comment (biwano) [inline carbon/app/[locale]/off-chain-vs-on-chain/page.tsx:19]: What does the z-index fixes?

File: package.json [hunk 2]
```diff
@@ -53,5 +53,8 @@
     "concurrently": "^7.6.0",
     "prettier": "3.0.2",
     "prettier-plugin-organize-imports": "^3.2.3"
+  },
+  "dependencies": {
+    "@emotion/css": "^11.11.2"
```
Comment (biwano) [inline package.json:58]: Do we really need this? If yes, I think it is better to add it in the carbon module package.json file.

**Review by biwano — APPROVED** [review]
Great Job! Those tootips are legendary.
Just nitpicks.

File: package.json (hunk 2 above)
Comment (sprrwhwk) [reply package.json]: We added it while updating the option switch components. I moved the dependency to the carbon sub module in [09c65dd](https://github.com/KlimaDAO/klimadao/pull/1644/commits/09c65ddd27071a7c065ec87a976b86774b427a58)

File: carbon/app/[locale]/off-chain-vs-on-chain/page.tsx (hunk 1 above)
Comment (sprrwhwk) [reply carbon/app/[locale]/off-chain-vs-on-chain/page.tsx:19]: it fixes the tooltips being shown behind oder cards:

grafik

**Pull request merged**